lib/acts-as-taggable-on-mongoid.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require "acts_as_taggable_on_mongoid/version"
+
+# The base module for the gem under which all classes are namespaced.
+module ActsAsTaggableOnMongoid
+  extend ::ActiveSupport::Autoload
+
+  eager_autoload do
+    autoload :Configuration
+    autoload :TagList
+    autoload :GenericParser
+    autoload :DefaultParser
+    # autoload :Tagger
+    # autoload :Tagging
+    # autoload :TagsHelper
+    # autoload :VERSION
```

Comment:
I left in a TON of comments of things I did not bring over from ActsAsTaggableOn.

I am currently somewhat conflicted on if I should take them out or not only because a lot of them should probably eventually come over..